mask2former/modeling/pixel_decoder/ops/setup.py

```diff
@@ -36,7 +36,8 @@ def get_extensions():
     extra_compile_args = {"cxx": []}
     define_macros = []
 
-    if torch.cuda.is_available() and CUDA_HOME is not None:
+    # Force cuda since torch ask for a device, not if cuda is in fact available.
+    if (os.environ.get('FORCE_CUDA') or torch.cuda.is_available()) and CUDA_HOME is not None:
```

Comment:
Please add "or setting FORCE_CUDA=1" to the error message in L54.

Reply:
Updated the error message